Sources/TimeFreeze.swift

```diff
@@ -22,6 +26,35 @@ struct TimeFreeze {
         self.uptime = TimeFreeze.systemUptime()
     }
 
+    init?(from dictionary: [String: TimeInterval]) {
```

Comment:
It's maybe not that big a deal, but it feels a teeny bit off to me that this takes a dictionary, but the constant keys required to be in it are private to this file.

What about simply having this take the 3 args, and defining the keys up with `loadFromDefaults`?

Reply:
I remembered why I didn't do this: TimeFreeze's ivars are private and we'd need to expose them if the logic in `TimeFreeze.toDictionary()` was moved up to `Clock`.